src/shared/components/case-editor/domain/event-completion-states.enum.model.ts

```diff
@@ -0,0 +1,12 @@
+export enum EventCompletionStates {
+  CHECK_TASKS_CAN_BE_COMPLETED = 'check-tasks-can-be-completed',
```

Comment:
Again here can we follow the typescript guide on `enums`.

Reply:
The state machine expects the text/value to be in the format _check-tasks-can-be-completed_, so I had to go with this.

Reply:
Is the text _CHECK_TASKS_CAN_BE_COMPLETED_ should be _CheckTasksCanBeCompleted_?